Rel v0.2.0

macpijan opened the pull request.

TomaszAIR commented:
@arturkow2000 I believe that README should also be changed. There are only informations about zephyr usage.

TomaszAIR commented on README.md:
```diff
@@ -2,25 +2,42 @@
 
-The [run.sh script](run.sh) will mount the zephyr directory inside the
-container automatically and initialize Zephyr for work. To build a sample
+The [run-zephyr.sh script](run-zephyr.sh) will mount the zephyr directory inside
```
At some point we probably drop that, good that you changed the name  for zephyr.

0xDen reviewed (approved):
LGTM.
I confirm. I successfully built demo project (blinky-demo-nrf52840) as it's describe in this branch.

macpijan committed:
*.sh: improve portability

/bin/bash is not perfectly portable (does not exists on NixOS, for
example). /usr/bin/env should be the best fit.

Signed-off-by: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>

macpijan committed:
Dockerfile: improve reproducibility

Signed-off-by: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>

macpijan committed:
dynamically create user

Signed-off-by: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>

macpijan commented:
@krystian-hebel Please try it out. It should dynamically create user with UID of current active user.

```
$ id -u
1000

$ run-fobnail-sdk.sh id -u
1000
```

krystian-hebel reviewed (approved):
Seems to be working, but I haven't tested case where image is built as one user and started as another one. Also, IIUC it still assumes that `fobnail0` TAP was created by running user on host, but perhaps this is an issue that has to be solved elsewhere.

macpijan commented:
> Also, IIUC it still assumes that fobnail0 TAP was created by running user on host, but perhaps this is an issue that has to be solved elsewhere.

Yes. Some attempt was started here, to simplify the setup for the TPM simulator, at least. https://github.com/fobnail/fobnail-attester/pull/15
